torch/_subclasses/fake_tensor.py

```diff
@@ -253,10 +253,13 @@ def from_real_tensor(
         source=None,
         dynamic_dims: Optional[DimList[DimDynamic]] = None,
         constraint_dims: Optional[DimList[DimConstraint]] = None,
+        memoized_only=False,
     ):
         maybe_memo = self._get_memo(t)
         if maybe_memo is not None:
             return maybe_memo
+        if memoized_only:
+            return None
```

Comment:
Better to raise an exception?

Reply:
I prefer None here. You asked for it via API, nothing is exceptional.